ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -144,6 +144,16 @@ public Account getAccountById(short id) {
     return accountInfoMapRef.get().getAccounts();
   }
 
+  @Override
+  public short getContainerIdForLegacyPutPublicBlob() {
```

Comment:
Removed these two methods in the interface. Now just use `Container#UNKNOWN_PUBLIC_CONTAINER_ID` and `Container#UNKNOWN_PRIVATE_CONTAINER_ID` for both `InMemoryUnknownAccountService` and `HelixAccountService`. There should be no case that [-1, 0] and [-1, 1] would be generated, as long as `HelixAccountService` is switched in before turning on BlobId and BlobProperties v2 write path.